jib-core/src/main/java/com/google/cloud/tools/jib/image/json/V22ManifestListTemplate.java

```diff
@@ -68,13 +70,32 @@
       "application/vnd.docker.distribution.manifest.list.v2+json";
   private static final int SCHEMA_VERSION = 2;
 
+  private final int schemaVersion = SCHEMA_VERSION;
+  private final String mediaType = MANIFEST_MEDIA_TYPE;
+
   @Override
   public int getSchemaVersion() {
-    return SCHEMA_VERSION;
+    return schemaVersion;
+  }
+
+  public String getMediaType() {
+    return mediaType;
   }
 
   @Nullable private List<ManifestDescriptorTemplate> manifests;
 
+  /**
+   * Adds a manifest to the manifestList.
+   *
+   * @param manifest a JSON representation of an {@link Image}
```

Comment:
This class shouldn't mention `Image`. The class is totally unrelated. I think we can just go with

```suggestion
   * @param manifest a manifest descriptor
```